### PR TITLE
Rewrite

### PR DESCRIPTION
**Pull request by yayuyokitano**
Some changes need to be done in settings.

Before merging this:
Settings -> Actions -> General -> Workflow Permissions -> Set to Read and Write Permissions.
This new version has a deploy action that writes the static page to a gh-pages branch, it needs perms for this.

After merging this and once initial deploy action completes:
Settings -> Pages -> Set source to Deploy from Branch
Set Branch to gh-pages and / (Root)
Custom domain should already be set.

We may also have to add a CNAME entry to the top level *in the gh-pages branch* once it is created. This should not be overwritten by the action according to docs, but I have not been able to test it.

Most up to date live deploy of this available on https://yayuyokitano.github.io/web-scrobbler.github.io/
However, it errors, as it expects to be deployed to top-level, which does not happen on the fork.
Here's an only slightly older version that does not error as it expects to be deployed on /wss https://yayuyokitano.github.io/wss/

### CODE REVIEW

**Review by inverse — COMMENTED** [review]
For some reason it errors for me when running `npm start`

image
 
Also I noticed some elements don't have spaces on my side, wonder what's causing that?

**Review by inverse — COMMENTED** [review]
For some reason it errors for me when running `npm start`

image

Do you think the red and white are too strong? e.g. for people who are visually impaired?  I think for the logo it's fine but for the whole header/footer it seems a bit too strong to me.

image
 
Also I noticed some elements don't have spaces on my side, wonder what's causing that?

File: .github/dependabot.yml
[the remark this answers is not in the document]
Comment (yayuyokitano) [reply .github/dependabot.yml:1]: Done

**Comment by yayuyokitano** [comment]
>Also I noticed some elements don't have spaces on my side, wonder what's causing that?

This was caused by a lack of &nbsp; and should be fixed on the new version. The older https://yayuyokitano.github.io/wss/ link does not have the fix.


>For some reason it errors for me when running npm start

Odd. I changed the recommended command, does this new command work?


>Do you think the red and white are too strong?

I agree. I darkened and desaturated it a bit, I think it's better now. What do you think?

**Comment by yayuyokitano** [comment]
That means it's a problem – I've removed it.

We can figure out a better background later, as I think not having it makes the design a little bit barren.

**Comment by yayuyokitano** [comment]
By the way – I don't have sufficient permissions for this repo – you will have to go through the steps to merge and deploy

**Comment by inverse** [comment]
made you org owner :+1:

**Comment by yayuyokitano** [comment]
Aight. Should I go through the steps to try to deploy this now despite the temporary rickroll on the safari link?
I would like to have this up first for purposes of getting-around-apple-restrictions-as-well-as-possible related things

**Comment by inverse** [comment]
I think people may find that funny :) Yeah those apple restrictions seem painful  :/

**Comment by yayuyokitano** [comment]
Aight lets hope for the best

**Comment by yayuyokitano** [comment]
Finished all the steps now, hopefully it goes up within the next couple minutes

**Comment by yayuyokitano** [comment]
And it's up!

**Pull request merged**